docs/DEVELOPMENT.md

```diff
@@ -2,31 +2,49 @@
 - make
 - golang 1.16+
 
-## Clone source code
-```
-git clone -b CCv0-peerpod https://github.com/yoheiueda/kata-containers.git
-git clone -b code-import https://github.com/yoheiueda/cloud-api-adaptor.git
+## Clone the source code
 ```
+export BUILD_DIR=$HOME/remote-hyp
+mkdir -p $BUILD_DIR && cd $BUILD_DIR
 
-## Build for AWS
-```
+git clone -b CCv0-peerpod https://github.com/yoheiueda/kata-containers.git
+git clone https://github.com/confidential-containers/cloud-api-adaptor.git
 cd cloud-api-adaptor
-CLOUD_PROVIDER=aws make
 ```
 
-## Build for IBMCloud
+## Build the binary
+
+Set `CLOUD_PROVIDER` variable to either `aws|ibmcloud|libvirt` depending on your requirement.
+
 ```
-cd cloud-api-adaptor
-CLOUD_PROVIDER=ibmcloud make
+export CLOUD_PROVIDER=aws
+make
 ```
 
-## Build for libvirt
 Note that libvirt go library uses `cgo` and hence there is no static build.
+
 Consequently you'll need to run the binary on the same OS/version where you have
 built it.
 You'll also need to install the libvirt dev packages before running the build.
 
+Example, if you are using Ubuntu then run the following command
 ```
-cd cloud-api-adaptor
-CLOUD_PROVIDER=libvirt make
+sudo apt-get install -y libvirt-dev
+```
+
+## Build Kata runtime
+
+Install the prerequisites as mentioned in the following [link](https://github.com/kata-containers/kata-containers/blob/main/docs/Developer-Guide.md#requirements-to-build-individual-components)
+
+Install `protoc`
+```
+wget -c https://github.com/protocolbuffers/protobuf/releases/download/v3.11.4/protoc-3.11.4-linux-x86_64.zip
+sudo unzip protoc-3.11.4-linux-x86_64.zip -d /usr/local
+```
+
+Build the runtime
+
+```
+cd $BUILD_DIR/kata-containers/runtime
```

Comment:
Should be $BUILD_DIR/kata-containers/**src**/runtime

Apart from that fix, this PR looks good to me. Thanks @bpradipt !

Reply:
Thanks @wainersm. Fixed it. PTAL